build/net/src/lib.rs

```diff
@@ -144,5 +194,16 @@ pub fn generate_socket_enum(
     }
     writeln!(out, "];")?;
 
+    writeln!(
+        out,
+        "#[allow(unused)]\
```

Comment:
The backslash causes this to be generated on the same line as the pub const, was that deliberate?

Reply:
"Intentional" is a strong word for it; all the other writes in this function also do it.